BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -296,6 +296,11 @@
 
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesBase  | 0x00000000
   gPlatformModuleTokenSpaceGuid.PcdSmbiosStringsPtr  | 0x00000000
+  gPlatformModuleTokenSpaceGuid.PcdP2sb0Ptr  | 0xFEF10EE0
```

Comment:
Same comment as in the DEC file. Core DEC/DSC may not be the right place for these PCDs.  Can these PCDs be declared in the /Silicon/CommonSocPkg/CommonSocPkg.dec file and initialized in BoardConfig.py?

Maybe PcdGpioDebugPortMmioBase could be an example.

Reply:
I've removed this since it will be update dynamically, thanks.